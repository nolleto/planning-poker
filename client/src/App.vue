<template>
  <div id="app">
    <nav-header/>
    <router-view/>
  </div>
</template>

<script>
import NavHeader from './components/NavHeader'

export default {
  components: {
    NavHeader
  }
}
</script>

<style>
body {
  margin: 0;
}
</style>

<style scoped>
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
}
</style>
