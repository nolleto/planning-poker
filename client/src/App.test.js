import App from './App.vue'
import { shallowMount } from '@vue/test-utils'
import Vue from 'vue'
import Router from 'vue-router'

const shallowComponent = () => {
  return shallowMount(App)
}

describe('App component', () => {
  let wrapper

  beforeAll(() => {
    Vue.use(Router)
  })

  describe('Snapshots', () => {
    beforeAll(() => {
      wrapper = shallowComponent()
    })

    it('matches', () => {
      expect(wrapper.element).toMatchSnapshot()
    })
  })
})
