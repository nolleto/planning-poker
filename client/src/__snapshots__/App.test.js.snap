// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component Snapshots matches 1`] = `
<div
  id="app"
>
  <navheader-stub />
   
  <router-view-stub
    name="default"
  />
</div>
`;
